data/fields/roof/height.json

```
@@ -0,0 +1,6 @@
+{
+    "key": "roof:height",
+    "minValue": 0,
+    "type": "number",
+    "label": "Roof Height from facade to top"
```

```suggestion
    "label": "Roof height from facade to top"
```

or even just … because the definition definition is something that the info-i can handle.
```suggestion
    "label": "Roof height"
```

What is the unit here? We should add this field to https://github.com/ideditor/schema-builder/pull/198 right?